### DISCUSSION TITLE
convert skoda mib2 high to seat version

### DISCUSSION THREAD

**Discussion by mrmibtinker** [Mr-MIBonk/M.I.B._More-Incredible-Bash#577] [General]
convert skoda mib2 high to seat version 

how can i do this and also how can i go back to skoda

**Comment by olegbzk** [comment]
The same. 
Want to convert Skoda to VW.
here link to [mibwiki](https://mibwiki.one/doc/between-brands-g1g44zDQBW) but it is down.

**Comment by mrga37** [comment]
@mrmibtinker Have you managed to do a conversion?
